plugins/ingest-attachment/build.gradle

```diff
@@ -23,7 +23,7 @@ esplugin {
 }
 
 versions << [
-  'tika': '1.17',
+  'tika': '1.18',
```

Comment:
should we add a note here about the discrepancy between ES's dependency on jackson, and tika's?

Reply:
Added.